tools/windows_debug_syms.pl

```diff
@@ -22,6 +22,9 @@
 	# Replace the .exe or .dll suffix of each file with .pdb
 	$line =~ s/\.(exe|dll)$/\.pdb/;
 	
+	# Ensure commercial engine .pdbs have the expected filename (i.e. "LiveCode-Commercial.pdb")
+	$line =~ s/(LiveCode-Business|LiveCode-Indy|LiveCode-CommunityPlus)\.pdb$/LiveCode-Commercial\.pdb/;
```

Comment:
So my perl is very rusty - what does this do exactly?

Reply:
My understanding is that this script creates the list of pdb files that need to be copied to the archive. The previous  line added (among others) the files `LiveCode-Business.pdb`, `LiveCode-Indy.pdb`, and `LiveCode-CommunityPlus.pdb`. But these 3 files do not exist - the actual file we need is the `LiveCode-Commercial.pdb` one.

So this line just replaces `LiveCode-Business.pdb`, `LiveCode-Indy.pdb`, and `LiveCode-CommunityPlus.pdb` with `LiveCode-Commercial.pdb` in this list.